### PR TITLE
Feature/pelagos 3710 change datepicker

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

**Comment by praneethpr** [comment]
Need to merge PELAGOS-3676

**Comment by fightingtexasaggie** [comment]
I'm not totally sure if this will be a problem, but sending a date in like 2012-12-01 works fine, but 2012-12-1 results in an exception being thrown by elastic.

**Comment by praneethpr** [comment]
> I'm not totally sure if this will be a problem, but sending a date in like 2012-12-01 works fine, but 2012-12-1 results in an exception being thrown by elastic.

I tried it, the client-side validation shouldn't let you submit the form. I think you were able to get past because the JS libraries weren't found. yarn encore dev should solve the issue

File: src/Pelagos/Bundle/AppBundle/Resources/public/js/search.js [hunk 1]
```diff
@@ -112,6 +113,54 @@ $(document).ready(function () {
         });
     });
 
+    // set up DatePickers
+    $("#collectionStartDate").datepicker({
+        //defaultDate: "",
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Resources/public/js/search.js]: comments.

File: src/Pelagos/Bundle/AppBundle/Resources/public/js/search.js [hunk 2]
```diff
@@ -112,6 +113,54 @@ $(document).ready(function () {
...
+            collectionStartDate: "Collection Start Date is not a valid ISO date",
+            collectionEndDate: "Collection End Date is not a valid ISO date",
+        },
+        ignore: ".ignore,.prototype",
+        submitHandler: function (form) {
+            if ($(".ignore").valid()) {
+                form.submit();
+            }
+        }
+
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Resources/public/js/search.js]: extra line?

**Pull request merged**